src/parser/shaman/elemental/modules/talents/CallTheThunder.js

```diff
@@ -65,20 +69,20 @@ class CallTheThunder extends Analyzer {
     }
   }
 
-  onEarthShockOrEarthquakeDamage(event) {
-    if (event.ability.guid === SPELLS.EARTH_SHOCK.id) {
-      this.earthShockDamage += event.amount;
-    } else if (event.ability.guid === SPELLS.EARTHQUAKE_DAMAGE.id) {
-      this.earthQuakeDamage += event.amount;
-    }
+  onEarthShockDamage(event) {
+    this.earthShockDamage += event.amount;
```

Comment:
This is my fault. I should have noted it by you should also include the absorbed damage or on fights like ashvane you could have 0s as the number.